tests/test_training_args.py

```diff
@@ -0,0 +1,113 @@
+from unittest import TestCase
+
+import pytest
+
+from setfit.training_args import TrainingArguments
+
+
+class TestTrainingArguments(TestCase):
+    def test_training_args_raises_error_with_wrong_warmup_proportion(self):
```

Comment:
Since your class is called `TestTrainingArguments`, I think you can remove `training_args` from the function names
```suggestion
    def test_raises_error_with_wrong_warmup_proportion(self):
```

Reply:
Done!